bump_version.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
```

Comment:
maybe put the script in .github/workflows or just write it plainly in the yaml file?
```
        run: |
          echo a
          echo b         
```

Reply:
Added straight to the yaml file, seems somewhat strange, but I guess it should work.